### ISSUE TITLE
/no/ssh/id/file/specified: No such file or directory

### ISSUE THREAD

**Issue by bphermansson** [aktos-io/dcs-tools#7]
Hi! 

I see this on my Raspbian system:

```
make backup-root
/no/ssh/id/file/specified: No such file or directory
/no/ssh/id/file/specified: No such file or directory
/no/ssh/id/file/specified: No such file or directory
project root is correct
/no/ssh/id/file/specified: No such file or directory
/no/ssh/id/file/specified: No such file or directory
/no/ssh/id/file/specified: No such file or directory
tor  9 mar 2017 18:38:24 CET
```

But the script seems to work. How do I get rid of that annoying message?

**Comment by ceremcem** [comment]
Source of this error (warning) message is [here](https://github.com/aktos-io/aktos-dcs-tools/blob/bbe3d3e246a0bfa2938b7d4c8d114e85ad7125fb/Makefile#L12). Toolset expects a valid ssh key path and this variable is overwritten in [this line](https://github.com/aktos-io/aktos-dcs-tools/blob/bbe3d3e246a0bfa2938b7d4c8d114e85ad7125fb/Makefile#L31) by `config.mk`. 

Just run `./aktos-dcs-tools/configure` again and uncomment [the line](https://github.com/aktos-io/aktos-dcs-tools/blob/master/default-config.mk#L7) and specifiy your ssh key file name (eg. `/home/pi/.ssh/id_rsa`).

**Comment by ceremcem** [comment]
This issue should be fixed by v2.0.

**Comment by ceremcem** [comment]
Closing due to timeout. Reopen this issue if needed.
